interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -266,24 +266,11 @@ function($log, $cacheFactory, $q, Sample, Activity) {
 function SampleBinCtrl($scope, $log, $uibModal, Sample, SampleBin) {
   // give our templates a way to access the SampleBin service
   $scope.sb = SampleBin;
-
-  $scope.show = function() {
-    $uibModal.open({
-      animation: true,
-      templateUrl: 'analyze/analysis/analysisModal.tpl.html',
-      controller: 'AnalysisModalCtrl',
-      size: 'lg',
-      resolve: {
-        analysis: function() {
-          return $scope.analysis;
-        }
-      }
-    });
-  };
 }])
 
 .directive('sampleBin', function() {
   return {
+    replace: true,
```

Comment:
Without this, the `<sample-bin>` tags are left in the DOM and it totally messes up the CSS styling for the menu. Took me a while to figure that one out!